### PR TITLE
WordCamp: Apply coding standards

### PR DESCRIPTION
**Pull request by vedanshujain**
- [x] PHP
- [ ] JavaScript

### CODE REVIEW

File: public_html/wp-content/mu-plugins/blocks/view/shared/grid-layout.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ function render_grid_layout( $layout, $columns, $children, $container_classes )
 	<ul class="<?php echo esc_attr( $container_classes ); ?>">
 		<?php foreach ( $children as $child ) { ?>
 			<li class="wordcamp-block-post-list-item wordcamp-grid-layout-item wordcamp-clearfix">
-				<?php echo $child ?>
+				<?php printf( wp_kses_post( $child ) ); ?>
```
Comment (vedanshujain) [inline public_html/wp-content/mu-plugins/blocks/view/shared/grid-layout.php]: - [ ] Make sure this renders properly

File: public_html/wp-content/plugins/wc-post-types/inc/rest-api.php [hunk 2]
```diff
@@ -159,10 +159,12 @@ function register_additional_rest_fields() {
...
 	$post        = (object) $post;
 	$avatar_urls = [];
+	$email = get_post_meta( $post->id, '_wcb_speaker_email', true );
+	$user_id = get_post_meta( $post->id, '_wcpt_user_id', true );
 
-	if ( $email = get_post_meta( $post->id, '_wcb_speaker_email', true ) ) {
+	if ( $email ) {
 		$avatar_urls = rest_get_avatar_urls( $email );
-	} elseif ( $user_id = get_post_meta( $post->id, '_wcpt_user_id', true ) ) {
+	} elseif ( $user_id ) {
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/inc/rest-api.php]: - [ ] Test if we can fetch avatar URLs properly. 
- [ ] Align variables on line 162 and 163

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 3]
```diff
@@ -116,40 +119,39 @@ function admin_menu() {
...
 		$levels = $this->get_sponsor_levels();
 		?>
 		<div class="wrap">
-			<?php screen_icon(); ?>
-			<h1><?php _e( 'Order Sponsor Levels', 'wordcamporg' ); ?></h1>
+			<h1><?php esc_html_e( 'Order Sponsor Levels', 'wordcamporg' ); ?></h1>
 
 			<?php if ( false !== $_REQUEST['updated'] ) : ?>
-				<div class="updated fade"><p><strong><?php _e( 'Options saved', 'wordcamporg' ); ?></strong></p></div>
+				<div class="updated fade"><p><strong><?php esc_html_e( 'Options saved', 'wordcamporg' ); ?></strong></p></div>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 4]
```diff
@@ -116,40 +119,39 @@ function admin_menu() {
...
 			<?php endif; ?>
 
 			<form method="post" action="options.php">
 				<?php settings_fields( 'wcb_sponsor_options' ); ?>
 				<div class="description sponsor-order-instructions">
-					<?php _e( 'Change the order of sponsor levels are displayed in the sponsors page template.', 'wordcamporg' ); ?>
+					<?php esc_html_e( 'Change the order of sponsor levels are displayed in the sponsors page template.', 'wordcamporg' ); ?>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 5]
```diff
@@ -160,7 +162,7 @@ function render_order_sponsor_levels() {
 				<?php endforeach; ?>
 				</ul>
 				<p class="submit">
-					<input type="submit" class="button-primary" value="<?php _e( 'Save Options', 'wordcamporg' ); ?>" />
+					<input type="submit" class="button-primary" value="<?php esc_attr_e( 'Save Options', 'wordcamporg' ); ?>" />
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 6]
```diff
@@ -436,7 +448,7 @@ function shortcode_speakers( $attr, $content ) {
 				?>
 
 				<!-- Organizers note: The id attribute is deprecated and only remains for backwards compatibility, please use the corresponding class to target individual speakers -->
-				<div id="wcorg-speaker-<?php echo sanitize_html_class( $post->post_name ); ?>" class="<?php echo implode( ' ', $speaker_classes ); ?>">
+				<div id="wcorg-speaker-<?php echo sanitize_html_class( $post->post_name ); ?>" class="<?php echo( esc_attr( implode( ' ', $speaker_classes ) ) ); ?>">
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 7]
```diff
@@ -838,24 +855,24 @@ function fav_session_share_form() {
...
+	 * @param int $speaker_id int The speaker's post id.
 	 *
 	 * @return NULL|string
 	 */
-	function get_speaker_wporg_permalink( $speaker_id ) {
+	public function get_speaker_wporg_permalink( $speaker_id ) {
 		$post = get_post( $speaker_id );
-		if ( $post->post_type != 'wcb_speaker' || $post->post_status != 'publish' ) {
-			return;
+		if ( 'wcb_speaker' != $post->post_type  || 'publish' != $post->post_status ) {
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ]. Test basic sanity flow

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 8]
```diff
@@ -1152,15 +1173,17 @@ function shortcode_sessions( $attr, $content ) {
 					}
 				}
 
-				// Gather data for list of links
-				if ( $url = get_post_meta( $post->ID, '_wcpt_session_slides', true ) ) {
+				// Gather data for list of links.
+				$url = get_post_meta( $post->ID, '_wcpt_session_slides', true );
+				if ( $url ) {
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Sanity test

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 9]
```diff
@@ -1152,15 +1173,17 @@ function shortcode_sessions( $attr, $content ) {
...
 					$links['slides'] = array(
 						'url'   => $url,
 						'label' => __( 'Slides', 'wordcamporg' ),
 					);
 				}
 
-				if ( $url = get_post_meta( $post->ID, '_wcpt_session_video', true ) ) {
+				$url = get_post_meta( $post->ID, '_wcpt_session_video', true );
+				if ( $url ) {
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Sanity test

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 10]
```diff
@@ -1176,8 +1199,8 @@ function shortcode_sessions( $attr, $content ) {
 
 					<div class="wcorg-session-description">
 						<?php the_post_thumbnail(); ?>
-						<?php echo $session_meta; ?>
-						<?php echo $speakers_avatars; ?>
+						<?php echo wp_kses_post( $session_meta ); ?>
+						<?php echo wp_kses_post( $speakers_avatars ); ?>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 11]
```diff
@@ -1645,12 +1681,12 @@ function metabox_speaker_info() {
 		<?php wp_nonce_field( 'edit-speaker-info', 'wcpt-meta-speaker-info' ); ?>
 
 		<p>
-			<label for="wcpt-gravatar-email"><?php _e( 'Gravatar Email:', 'wordcamporg' ); ?></label>
+			<label for="wcpt-gravatar-email"><?php esc_html_e( 'Gravatar Email:', 'wordcamporg' ); ?></label>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 12]
```diff
@@ -1645,12 +1681,12 @@ function metabox_speaker_info() {
...
 			<input type="text" class="widefat" id="wcpt-gravatar-email" name="wcpt-gravatar-email" value="<?php echo esc_attr( $email ); ?>" />
 		</p>
 
 		<p>
-			<label for="wcpt-wporg-username"><?php _e( 'WordPress.org Username:', 'wordcamporg' ); ?></label>
+			<label for="wcpt-wporg-username"><?php esc_html_e( 'WordPress.org Username:', 'wordcamporg' ); ?></label>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 13]
```diff
@@ -1675,7 +1711,7 @@ function metabox_organizer_info() {
 		<?php wp_nonce_field( 'edit-organizer-info', 'wcpt-meta-organizer-info' ); ?>
 
 		<p>
-			<label for="wcpt-wporg-username"><?php _e( 'WordPress.org Username:', 'wordcamporg' ); ?></label>
+			<label for="wcpt-wporg-username"><?php esc_html_e( 'WordPress.org Username:', 'wordcamporg' ); ?></label>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 14]
```diff
@@ -1716,7 +1752,7 @@ function metabox_speakers_list() {
 		?></textarea>
 
 		<p class="description">
-			<?php _e( 'A speaker entry must exist first. Separate multiple speakers with commas.', 'wordcamporg' ); ?>
+			<?php esc_html_e( 'A speaker entry must exist first. Separate multiple speakers with commas.', 'wordcamporg' ); ?>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 15]
```diff
@@ -1797,47 +1836,47 @@ function metabox_session_info() {
 		<?php wp_nonce_field( 'edit-session-info', 'wcpt-meta-session-info' ); ?>
 
 		<p>
-			<label for="wcpt-session-date"><?php _e( 'Date:', 'wordcamporg' ); ?></label>
+			<label for="wcpt-session-date"><?php esc_html_e( 'Date:', 'wordcamporg' ); ?></label>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 16]
```diff
@@ -1797,47 +1836,47 @@ function metabox_session_info() {
...
 			<input type="text" id="wcpt-session-date" data-date="<?php echo esc_attr( $session_date ); ?>" name="wcpt-session-date" value="<?php echo esc_attr( $session_date ); ?>" /><br />
-			<label><?php _e( 'Time:', 'wordcamporg' ); ?></label>
+			<label><?php esc_html_e( 'Time:', 'wordcamporg' ); ?></label>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 17]
```diff
@@ -1797,47 +1836,47 @@ function metabox_session_info() {
...
 
-			<select name="wcpt-session-hour" aria-label="<?php _e( 'Session Start Hour', 'wordcamporg' ); ?>">
+			<select name="wcpt-session-hour" aria-label="<?php esc_html_e( 'Session Start Hour', 'wordcamporg' ); ?>">
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 18]
```diff
@@ -1797,47 +1836,47 @@ function metabox_session_info() {
...
 				<?php for ( $i = 1; $i <= 12; $i++ ) : ?>
 					<option value="<?php echo esc_attr( $i ); ?>" <?php selected( $i, $session_hours ); ?>>
 						<?php echo esc_html( $i ); ?>
 					</option>
 				<?php endfor; ?>
 			</select> :
 
-			<select name="wcpt-session-minutes" aria-label="<?php _e( 'Session Start Minutes', 'wordcamporg' ); ?>">
+			<select name="wcpt-session-minutes" aria-label="<?php esc_html_e( 'Session Start Minutes', 'wordcamporg' ); ?>">
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 19]
```diff
@@ -1797,47 +1836,47 @@ function metabox_session_info() {
...
 				<?php for ( $i = '00'; (int) $i <= 55; $i = sprintf( '%02d', (int) $i + 5 ) ) : ?>
 					<option value="<?php echo esc_attr( $i ); ?>" <?php selected( $i, $session_minutes ); ?>>
 						<?php echo esc_html( $i ); ?>
 					</option>
 				<?php endfor; ?>
 			</select>
 
-			<select name="wcpt-session-meridiem" aria-label="<?php _e( 'Session Meridiem', 'wordcamporg' ); ?>">
+			<select name="wcpt-session-meridiem" aria-label="<?php esc_html_e( 'Session Meridiem', 'wordcamporg' ); ?>">
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/wc-post-types.php [hunk 20]
```diff
@@ -1797,47 +1836,47 @@ function metabox_session_info() {
...
 
 		<p>
-			<label for="wcpt-session-slides"><?php _e( 'Slides URL:', 'wordcamporg' ); ?></label>
+			<label for="wcpt-session-slides"><?php esc_html_e( 'Slides URL:', 'wordcamporg' ); ?></label>
 			<input type="text" class="widefat" id="wcpt-session-slides" name="wcpt-session-slides" value="<?php echo esc_url( $session_slides ); ?>" />
 		</p>
 
 		<p>
-			<label for="wcpt-session-video"><?php _e( 'WordPress.TV URL:', 'wordcamporg' ); ?></label>
+			<label for="wcpt-session-video"><?php esc_html_e( 'WordPress.TV URL:', 'wordcamporg' ); ?></label>
```
Comment (vedanshujain) [inline public_html/wp-content/plugins/wc-post-types/wc-post-types.php]: - [ ] Test if renders properly

File: public_html/wp-content/plugins/wc-post-types/inc/rest-api.php [hunk 21]
```diff
@@ -187,8 +189,8 @@ function get_avatar_urls_from_username_email( $post ) {
  */
 function register_fav_sessions_email() {
 	register_rest_route(
-		'wc-post-types/v1',     // REST namespace + API version
-		'/email-fav-sessions/', // URL slug
+		'wc-post-types/v1',     // REST namespace + API version.
+		'/email-fav-sessions/', // URL slug.
```
Comment (iandunn) [inline public_html/wp-content/plugins/wc-post-types/inc/rest-api.php]: I think we can just remove those comments, they seem unnecessary.

**Comment by vedanshujain** [comment]
Closing this now. Will add a  new one for JS

**Pull request closed without merging**